components/autoComplete/autoCompleteItem.js

```diff
@@ -29,13 +30,19 @@ class AutoCompleteItem extends Component {
     const {
       children,
       dataAttrs = {},
-      onClick,
       isActive,
       isTitle,
-      ...otherProps
     } = this.props;
     const mods = this.props.mods ? this.props.mods.slice() : [];
 
+    const otherProps = omit(this.props, [
```

Comment:
maybe it will not have a huge impact, but what if move this array to top scope and don't create it for each component render?

Reply:
When an array is used on a specific function and that scope ends, it gets cleaned by the GC.

What happens when it's global? 😉  But nevertheless, as you said, shouldn't be a problem